src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2946,6 +2946,12 @@ void SpellMgr::LoadSpellInfoCustomAttributes()
         }
     }
 
+    // add attribute custom to liquid auras
+    for (LiquidTypeEntry const* liquid : sLiquidTypeStore)
+        if (liquid->SpellId)
```

Comment:
We use enclosing braces `{ }` following a for/if whenever the next instruction is a block

Reply:
Ready, thanks